### ISSUE TITLE
new GTDB databases for rs220: draft

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash#3246]
**Note: not yet final**

release rs220 from GTDB: https://data.ace.uq.edu.au/public/gtdb/data/releases/release220/

sourmash databases constructed by @ccbaumler 🎉 using [directsketch](https://github.com/sourmash-bio/sourmash_plugin_directsketch) -

* [gtdb-rs220-k21.zip](https://farm.cse.ucdavis.edu/~ctbrown/sourmash-db/gtdb-rs220/gtdb-rs220-k21.zip) - 17 GB
* [gtdb-rs220-k31.zip](https://farm.cse.ucdavis.edu/~ctbrown/sourmash-db/gtdb-rs220/gtdb-rs220-k31.zip) - 17 GB
* [gtdb-rs220-k51.zip](https://farm.cse.ucdavis.edu/~ctbrown/sourmash-db/gtdb-rs220/gtdb-rs220-k51.zip) - 17 GB
* [gtdb-rs220.lineages.csv.gz](https://farm.cse.ucdavis.edu/~ctbrown/sourmash-db/gtdb-rs220/gtdb-rs220.lineages.csv.gz) - 13 MB
 
**Note:** The databases are missing approximately 200 genomes that have been deprecated/suspended by GenBank.

## Questions/thoughts

- should we release a higher scaled / smaller download?
- probably no need for an SBT or LCA any more
- what do we do about the missing genomes?
- do we remove the missing genomes from the lineages file?

Related issues:
* https://github.com/sourmash-bio/sourmash/issues/3183

**Comment by nmb85** [comment]
Are there sketches of a reduced genome-representatives version of the GTDB rs220 release? Not complaining if not. Thank you, @ccbaumler, for what you've provided here!

**Comment by ccbaumler** [comment]
Yup! Representative genomes have been sketched. I will be meeting with @ctb next week to get everything set up.

**Comment by nmb85** [comment]
Happy New Year! Just wondering if these databases have been posted publicly.

**Comment by ctb** [comment]
they were not available for download because of a broken configuration/setup on our end - I think it's fixed. We still haven't posted them to the databases page but they are available via the links at the top of this issue.

**Comment by nmb85** [comment]
Thank you very much!
